### PR TITLE
Add CAEN v895 GUI

### PR DESCRIPTION
**Pull request by FreddieAkeroyd**
### Description of work

Add GUI for CAEN v895

### Ticket

See ISISComputingGroup/IBEX#4915

### Acceptance criteria

*List the acceptance criteria for the PR. The aim is provide information to help the reviewer*

### Unit tests

*Give an overview of unit tests you have added or modified, if applicable. The aim is provide information to help the reviewer*

### System tests

*Mention any automated tests or manual tests that you have added or modified, if applicable. The aim is provide information to help the reviewer*

### Documentation
*Highlight and provide a link to any additions or changes to the documentation, if applicable. The aim is provide information to help the reviewer*

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

**Review by JamesKingWork — CHANGES_REQUESTED** [review]
Mostly good, a couple of usability issues

**Comment by Tom-Willemsen** [comment]
Noticed by @kjwoodsISIS : the font on the OPI is too small to be readable. Can we increase it's size please?

Compare with eurotherm OPI:
image

File: base/uk.ac.stfc.isis.ibex.opis/resources/CAENv895.opi [hunk 1]
```diff
@@ -0,0 +1,969 @@
...
+    <tab_6_font>
+      <opifont.name fontName="Segoe UI" height="9" style="0" pixels="false">ISIS_TabTitle_NEW</opifont.name>
+    </tab_6_font>
+    <tab_6_foreground_color>
+      <color name="ISIS_Standard_Text" red="0" green="0" blue="0"/>
+    </tab_6_foreground_color>
+    <tab_6_icon_path/>
+    <tab_6_title>Card 6</tab_6_title>
+    <tab_count>7</tab_count>
+    <tooltip>Disabled tabs are because Eurotherm is not active in IOC.</tooltip>
```
[the remark this answers is not in the document]
Comment (John-Holt-Tessella) [reply base/uk.ac.stfc.isis.ibex.opis/resources/CAENv895.opi:153]: Can not see a script to disable tabs

**Comment by John-Holt-Tessella** [comment]
Hopefully better

**Review by JamesKingWork — APPROVED** [review]

**Pull request merged**